packages/d3fc-annotation/src/canvas/band.js

```diff
@@ -0,0 +1,101 @@
+import { scaleIdentity } from 'd3-scale';
+import constant from '../constant';
+
+export default () => {
+
+    let xScale = scaleIdentity();
+    let yScale = scaleIdentity();
+    let orient = 'horizontal';
+    let fromValue = d => d.from;
+    let toValue = d => d.to;
+    let decorate = () => {};
+    let context = null;
+
+    var instance = (data) => {
+
+        if (orient !== 'horizontal' && orient !== 'vertical') {
+            throw new Error('Invalid orientation');
+        }
+
+        if (context === null) {
+            throw new Error('Context is not defined');
+        }
+
+        const horizontal = orient === 'horizontal';
+        // the value scale which the annotation 'value' relates to, the crossScale
+        // is the other. Which is which depends on the orienation!
+        const crossScale = horizontal ? xScale : yScale;
+        const valueScale = horizontal ? yScale : xScale;
+        const crossScaleRange = crossScale.range();
+
+        data.forEach((d, i) => {
+            context.save();
+            context.beginPath();
+            context.strokeStyle = 'transparent';
+
+            decorate(context, d, i);
+            const x = horizontal ? crossScaleRange[0] : valueScale(fromValue(d));
```

Comment:
As discussed, Ronan will look into whether it makes sense to use the `band` shape from d3fc-shapes.